blueman/main/NetConf.py

```diff
@@ -64,25 +64,27 @@ def do_apply(self):
                 self.do_remove()
 
             start, end = calc_ip_range(self.netconf.ip4_address)
-            args = "--pid-file=/var/run/dnsmasq.pan1.pid --bind-interfaces --dhcp-range=%s,%s,60m" \
-                " --except-interface=lo --interface=pan1 --dhcp-option=option:router,%s" % \
-                (inet_ntoa(start), inet_ntoa(end), inet_ntoa(self.netconf.ip4_address))
+            cmd = [have("dnsmasq"), "--pid-file=/var/run/dnsmasq.pan1.pid", "--except-interface=lo",
+                   "--interface=pan1", "--bind-interfaces", "--except-interface=lo",
```

Comment:
`--except-interface=lo` is included twice.

Reply:
Thanks, fixed.